common/display/display.cpp

```diff
@@ -176,6 +176,11 @@ bool Display::GetDisplayName(uint32_t *size, char *name) {
   return true;
 }
 
+bool Display::GetDisplayPipe(uint32_t *pipe_id) {
```

Comment:
In Test section, can u add what test needs to be run and expected results. Also, can u add a Jira for this feature and use that for Jira here and in u other patch